rdr_service/cron_default.yaml

```diff
@@ -89,16 +89,6 @@ cron:
   timezone: America/New_York
   schedule: every monday 05:30
   target: offline
-- description: Genomic Reconciliation Array Data Workflow
-  url: /offline/GenomicArrayReconciliationWorkflow
-  timezone: America/New_York
-  schedule: every day 05:30
-  target: offline
-- description: Genomic Reconciliation WGS Data Workflow
-  url: /offline/GenomicWGSReconciliationWorkflow
-  timezone: America/New_York
-  schedule: every day 05:45
-  target: offline
```

Comment:
If we are removing these jobs, we need to clean the repo of the logic that coincides with this also.